libs/application/templates/parental-leave/src/fields/Review/ReadOnlyReview.tsx

```diff
@@ -0,0 +1,337 @@
+                <GridColumn span="6/12">
+                  <Text>
+                    {
+                      getValueViaPath(
+                        application.answers,
+                        'payments.pensionFund',
+                      ) as string[]
+                    }
+                  </Text>
+                </GridColumn>
+                <GridColumn span="6/12">
+                  <Text>
+                    {
+                      getValueViaPath(
+                        application.answers,
+                        'payments.union',
+                      ) as string[]
+                    }
+                  </Text>
+                </GridColumn>
+              </GridRow>
+              <Box marginTop={3} />
+              <GridRow>
+                <GridColumn span="12/12">
+                  <Box marginTop={1} marginBottom={2}>
+                    <Text variant="h5">
+                      {formatMessage(m.privatePensionFundName)}
+                    </Text>
+                  </Box>
+                  <Text>
+                    {
+                      getValueViaPath(
+                        application.answers,
+                        'usePrivatePensionFund',
+                      ) as string[]
+                    }
+                  </Text>
+                </GridColumn>
+              </GridRow>
+
+              {statefulPrivatePension === YES && (
+                <>
+                  <Box marginTop={3} />
+                  <GridRow>
+                    <GridColumn span="6/12">
+                      <Text>
+                        {
+                          getValueViaPath(
+                            application.answers,
+                            'payments.pensionFund',
+                          ) as string[]
+                        }
+                      </Text>
+                    </GridColumn>
+                    <GridColumn span="6/12">
+                      <Text>
+                        {
+                          getValueViaPath(
+                            application.answers,
+                            'payments.union',
+                          ) as string[]
+                        }
+                      </Text>
+                    </GridColumn>
+                  </GridRow>
+                </>
+              )}
+            </Box>
+          </AccordionItem>
+          <AccordionItem
+            id="id_1"
+            label={formatMessage(mm.employer.subSection)}
+            startExpanded={allItemsExpanded}
+          >
+            <Box paddingY={4}>
+              <GridRow>
+                <GridColumn span="12/12">
+                  <Text>
+                    {
+                      getValueViaPath(
+                        application.answers,
+                        'employer.email',
+                      ) as string[]
+                    }
+                  </Text>
+                </GridColumn>
+              </GridRow>
+            </Box>
```

Comment:
Same as the last comment regarding unnecessary markup, this applies to all instances where we have a grid row with a single column with a span of 1 
```suggestion
            <Box paddingY={4}>
                  <Text>
                    {
                      getValueViaPath(
                        application.answers,
                        'employer.email',
                      ) as string[]
                    }
                  </Text>
            </Box>
```

Reply:
Same here, now refactored.